integrations/groups-taxonomy.php

```diff
@@ -102,10 +102,11 @@ function dt_push_groups() {
 				$groups_pushing = array( $groups_pushing );
 			}
 
-			$succeeded_groups = get_post_meta( $post->ID, 'dt_connection_groups_pushed', true );
+			$groups_pushed    = get_post_meta( $post->ID, 'dt_connection_groups_pushed', true );
+			$succeeded_groups = array();
 
-			if ( empty( $successed_groups ) || null === $succeeded_groups ) {
-				$succeeded_groups = array();
+			if ( empty( $groups_pushed ) ) {
```

Comment:
@armdev17 move this to line 105

Reply:
@arsendovlatyan I meant to check it the following way, for the cases when meta value is null or empty sting:
if ( ! $groups_pushed )

Otherwise, we can remove this condition, as after moving to line 105 it's value will be re-assigned anyway.